README.md

```diff
@@ -1,95 +1,69 @@
+This Git repository uses the [Textile](https://redcloth.org/textile) format with [Nanoc](http://nanoc.stoneship.org/) used to build the static site from the `master` branch.
 
-## Viewing the documentation
+## Running locally
 
-The static site generated from this documentation repository is hosted at <http://docs.ably.io> and deployed automatically when the `master` branch is updated.  We frequently publish updates from this repository so is typically more up to date than the [official Ably API documentation](https://www.ably.io/documentation).
+1. Clone a local version of the repo, or a local version of your forked repo.
+2. Checkout the master branch.
+3. Create and checkout your feature or fix branch.
 
-The official complete Ably documentation that incoporates all the documentation in this repository can be found at <https://www.ably.io/documentation>
+At this point you can either run the repo using Docker or directly with Ruby.
 
-## Deploying to website (www.ably.io/documentation)
+### Running with Docker
 
-The website consumes this repo through a ruby gem. The gem points at the `master` branch and saves the revision as a version. To release to `/documentation`, follow these steps:
+Using Docker, run the following command:
 
-1. Check if the changes that are already on `master` in this repository are ok to be released
-1. Merge your PR
-1. Clone the [website repo](https://github.com/ably/website)
-1. Run `bundle update ably-docs` in the website repo
-1. Check in `Gemfile.lock` that the SHA saved by the above command is the SHA of the commit you expected (in most cases this should be the merge/top commit of your PR)
-1. Open PR & ask the website team to review
-1. Follow the deployment process for website (if in doubt, ask the website team for help)
+* `docker-compose up`
 
-## Publishing to JSBin
+This starts up a web server at <http://localhost:4000>, builds the static site and allows you to view changes live.
 
-This configuration is not required for basic documentation creation and modification, so most
-editors can simply skip to
-[Forking and running locally](#forking-and-running-locally).
+_This has been tested with Docker version 2.3.0.3. See [Docker installation instructions](https://docs.docker.com/get-docker/)._
 
-If you will be creating or modifying code within this repository that is to be uploaded automatically
-to [JSBin](https://jsbin.ably.io/), for our "Try it now code editor",
-then you will need to create yourself a JSBin config file:
+_Note: On Windows you will need to re-run the `docker-compose up` command to see changes, unless you are using [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10)._
 
-    cp config/jsbin_config.example.yaml config/jsbin_config.yaml
+### Running with Ruby
 
-And then you will need to populate the `api_key` property in that file with a JSBin API key.
+Using Ruby directly (2.7.0) you must first run the following command:
```

Comment:
I would avoid specifying the Ruby version here, and instead link to the `.ruby_version` file stating that version should be used